### ISSUE TITLE
Additional stream operators to include in overview 

### ISSUE THREAD

**Issue by raboof** [akka/akka#25467]
There are a number of stream operators not yet documented at https://doc.akka.io/docs/akka/current/stream/operators/index.html .

The most current list of not-yet-documented operators can be found at https://github.com/akka/akka/blob/master/project/StreamOperatorsIndexGenerator.scala#L78 . Some probably can remain undocumented, those should be moved to the 'ignore list at https://github.com/akka/akka/blob/master/project/StreamOperatorsIndexGenerator.scala#L121

To document an operator:
* Optionally, add a comment to this issue 'claiming' the operator
* Remove it from the 'pending' list
* Add the markdown file to `/akka-docs/src/main/paradox/stream/operators`. The convention is `Classname/operatorname.md`, with the exception of operators that are available on both `Source`s and `Flow`s. Look at the other files for examples.
* run `sbt paradox` to test generating the documentation

**Comment by hamednourhani** [comment]
Hi @raboof is this issue still open for contribution ?

**Comment by raboof** [comment]
Yes, definitely! Also, it is completely fine to open up several small PR's that add single operators, it is by no means necessary to add them all at once.

**Comment by mahekchheda** [comment]
Hi @raboof, @hamednourhani  is this up for grabs? I would like to start contributing to this project with this issue.

**Comment by raboof** [comment]
@mahekchheda yes! It might be best to pick this up as several small PR's. That way each of them can be quickly reviewed and merged, and the risk of duplicated effort is reduced.

**Comment by ticofab** [comment]
Hey @raboof I'm giving a go at `Sink.collection` :) Is it correct that this operator is only available in the Scala API, with no equivalent in the Java API?

**Comment by raboof** [comment]
@ticofab that's great - yeah `Sink.collection` is only relevant for Scala as it uses the `CanBuildFrom` approach, you could direct Java API users to `Sink.seq`.

**Comment by raboof** [comment]
Or perhaps we should just introduce `Sink.collection` to the Java API and make it do something similar to `Sink.seq`... not sure ;)
